src/app/components/cases/CaseTimeline/components/TimelineEventItem.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 import { getDay }from "app/components/shared/DayDisplay/DayDisplay"
 
-import { Timeline } from "app/components/shared/Timeline"
+import Timeline from "@amsterdam/wonen-ui/components/timeline/Timeline/Timeline"
```

Comment:
This is also exported here, right? https://github.com/Amsterdam/wonen-ui/blob/main/src/index.ts
So could be `import Timeline from "@amsterdam/wonen-ui"` then.

Reply:
Then I get this error: "JSX element type 'Timeline' does not have any construct or call signatures.ts(2604)"